<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationDialog</class>
 <widget class="QDialog" name="AnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animation</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabControls">
      <attribute name="title">
       <string>Controls</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="5" column="0" colspan="4">
        <widget class="QComboBox" name="comboBoxTimePoints"/>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QLabel" name="labelNumberOfSteps">
         <property name="text">
          <string>Number of steps</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QLabel" name="labelAnimationType">
         <property name="text">
          <string>Animation type</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QLabel" name="labelTimePoints">
         <property name="text">
          <string>Set to time:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QComboBox" name="comboBoxAnimationType">
         <item>
          <property name="text">
           <string>time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>viewpoint</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelCurrentTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelCurrent">
         <property name="text">
          <string>Current time:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonRun">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonAdvance">
           <property name="text">
            <string>Advance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonStop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonSet">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QSpinBox" name="spinBoxNumberOfSteps"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDelay">
         <property name="text">
          <string>Delay:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxViewpoint">
         <property name="title">
          <string>Viewpoint</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelRotate">
            <property name="text">
             <string>Rotate:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditRotate"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelRotateUnits">
            <property name="text">
             <string>deg</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelElevate">
            <property name="text">
             <string>Elevate:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditElevate"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelElevateUnits">
            <property name="text">
             <string>deg</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelDelayUnits">
         <property name="text">
          <string>sec</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDelay"/>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBoxTimePoints</tabstop>
  <tabstop>comboBoxAnimationType</tabstop>
  <tabstop>lineEditDelay</tabstop>
  <tabstop>lineEditRotate</tabstop>
  <tabstop>lineEditElevate</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
